### PR TITLE
Update schema exploration to support non-default rp

### PR DESCRIPTION
**Pull request by goller**
- [x] CHANGELOG.md updated
  - [x] Rebased/mergable
  - [x] Tests pass
  - [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

Connect #979 

### The problem
The `SHOW TAG` queries did not use retentionPolicies so the schema explorer would have empty tags.

### The Solution
Add retentionPolicies to 'em

However, I'm pretty sure `SHOW MEASUREMENTS` isn't working like we want it to work.  It seems to show all measurements rather than just those for a specific retention policy.

### CODE REVIEW

File: ui/src/shared/apis/metaQuery.js [hunk 1]
```diff
@@ -36,14 +36,14 @@ export function showMeasurements(source, db) {
...
-  const query = `SHOW TAG KEYS FROM "${measurement}"`
+  const query = `SHOW TAG KEYS FROM "${retentionPolicy}"."${measurement}"`
 
   return proxy({source, db: database, rp: retentionPolicy, query})
 }
 
 export function showTagValues({source, database, retentionPolicy, measurement, tagKeys}) {
   const keys = tagKeys.sort().map((k) => `"${k}"`).join(', ')
-  const query = `SHOW TAG VALUES FROM "${measurement}" WITH KEY IN (${keys})`
+  const query = `SHOW TAG VALUES FROM "${retentionPolicy}"."${measurement}" WITH KEY IN (${keys})`
```
Comment (lukevmorris) [inline ui/src/shared/apis/metaQuery.js]: Is `retentionPolicy` guaranteed to be present in both cases?

Comment (jademcgough) [inline ui/src/shared/apis/metaQuery.js]: I think it's possible for `retentionPolicy` to be null.

Comment (timraymond) [inline ui/src/shared/apis/metaQuery.js]: If you fully qualify the query, like `SHOW TAG VALUES FROM "${database}"."${retentionPolicy}"."${measurement}" WITH KEY IN (${keys})`, then a null value for `retentionPolicy` yields a legit query. The construction `"database".""."measurement"` means "use the default RP.

Comment (goller) [inline ui/src/shared/apis/metaQuery.js]: Yup, @jademcgough you're right.  I'm following what you mentioned in slack and will try to use lodash.toString!

Comment (goller) [inline ui/src/shared/apis/metaQuery.js]: Ok! @lukevmorris @jademcgough @timraymond I've added checks for null RP here:

https://github.com/influxdata/chronograf/pull/1182/commits/e3c07e1797872fcdb6233e2676d8a4caa3dbde80

**Pull request merged**